README.md

```diff
@@ -283,6 +284,13 @@ or `py`.
 
 See the man page (`man fzf`) for the full list of options.
 
+#### Demo
+If you learn by watching videos, checkout this screencast to explore `fzf` features.
```

Comment:
- `checkout` -> `check out`
- `this screencast` -> `this screencast by [@samoshkin](https://github.com/samoshkin)`

Reply:
Ok, done.